microshift_release_notes/microshift-4-15-release-notes.adoc

```diff
@@ -278,4 +278,13 @@ Issued: 2024-04-16
 
 {product-title} release 4.15.9 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:1772[RHBA-2024:1772] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:1770[RHSA-2024:1770] advisory.
 
+For the latest images included with {microshift-short}, view the contents of the `microshift-release-info` RPM. See xref:../microshift_install/microshift-embed-in-rpm-ostree-offline-use.adoc#microshift-embed-microshift-image-offline-deploy_microshift-embed-in-rpm-ostree-offline-use[Embedding {microshift-short} containers for offline deployments].
```

Comment:
```suggestion
For the latest images included with {microshift-short}, view the contents of the `microshift-release-info` RPM. See xref:../microshift_install/microshift-embed-in-rpm-ostree-offline-use.adoc#microshift-embed-microshift-image-offline-deployment_microshift-embed-in-rpm-ostree-offline-use[Embedding {microshift-short} containers for offline deployments].
```

Reply:
This change is out of scope of this PR.

Reply:
@aravipra The link in the release notes is incorrect, the module name is wrong; so the link goes to the top of the page and not the module.

Reply:
@mburke5678 Thank you for pointing this out. 
Since this errata is dropped for this release, I am closing this PR. The suggested change is made in the jira [OSDOCS-10444](https://issues.redhat.com/browse/OSDOCS-10444).